File: sorting/bubble_sort/bubble_sort.js
const numbers = [9,4,1,2,0,8]

let j,i

function sort(arr){
    for(i=0; i< arr.length; i++){
        for(j=0; j<arr.length-i-1; j++){
            if(arr[j]>arr[j+1]){
                var temp
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
}

sort(numbers)

console.log(numbers)

File: sorting/bubble_sort/bubble_sort.py
numbers = [7,5,2,1,3]

def sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(n-i-1):
            if(arr[j]>arr[j+1]):
                arr[j], arr[j+1] = arr[j+1],arr[j]

sort(numbers)

print(numbers)